src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"></form>
```

Comment:
all inputs, fieldsets should be inside the form tag, otherwise it wouldn't work

```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
  ...put here all fieldsets
    </form>
```